suite/tests/api/ir_x86.c

```diff
@@ -737,6 +737,22 @@ test_nop_xchg(void *dc)
 #endif
 }
 
+static void
+test_hint_nops(void *dc)
+{
+    byte *pc;
+    instr_t *instr;
+    /* ensure we treat as nop. */
+    buf[0] = 0x0f;
+    buf[1] = 0x18;
+    buf[2] = 0x38;
```

Comment:
How about an array of these { { 0x0f, 0x18, 0x38 }, {...}} to cover all 4 variants?